includes/view/ShiftCalendarShiftRenderer.php

```diff
@@ -9,7 +9,6 @@
 use Engelsystem\Models\Shifts\ShiftSignupStatus;
 use Engelsystem\Models\User\User;
 use Illuminate\Support\Collection;
-
```

Comment:
This one is important says the linter:

` 11 | ERROR | [x] Header blocks must be separated by a single blank line`